lte/gateway/deploy/agw_install.sh

```diff
@@ -106,6 +106,7 @@ if [ "$MAGMA_INSTALLED" != "$SUCCESS_MESSAGE" ]; then
   apt-get -y install curl make virtualenv zip rsync git software-properties-common python3-pip python-dev
   alias python=python3
   pip3 install ansible
+  ansible-galaxy collection install community.general
```

Comment:
Let's wrap ansivle-galaxy collection install in the base role of agw installation.